kolibri/plugins/slideshow_render/assets/src/views/SlideshowRendererComponent.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+
+  <CoreFullscreen
+    ref="slideshowRenderer"
+    class="slideshow-renderer"
+    @changeFullscreen="isInFullscreen = $event"
+  >
+    <UiIconButton
+      class="btn"
+      :ariaLabel="isInFullscreen ? $tr('exitFullscreen') : $tr('enterFullscreen')"
+      color="primary"
+      size="small"
+      @click="$refs.slideshowRenderer.toggleFullscreen()"
+    >
+      <mat-svg v-if="isInFullscreen" name="fullscreen_exit" category="navigation" />
+      <mat-svg v-else name="fullscreen" category="navigation" />
+    </UiIconButton>
+    <Hooper v-if="slides" @slide="handleSlide">
+      <Slide v-for="slide in slides" :key="slide.id">
+        <div
+          class="slideshow-slide-image"
+          :style="{
+            height: `calc(100% - ${caption_height}px)`
+          }"
+        >
+          <img
+            :src="slide.storage_url"
+            :aria-labelledby="slideTextId(slide.id)"
+          >
+        </div>
+        <div :id="slideTextId(slide.id)" class="hidden-descriptive-text">
+          {{ slide.descriptive_text || slide.caption }}
+        </div>
+        <div :ref="slide.id" class="caption">
+          {{ slide.caption }}
+        </div>
+      </Slide>
+      <HooperNavigation
+        slot="hooper-addons"
+        :class="{'hooper-navigation-fullscreen' : isInFullscreen}"
+      />
+      <HooperPagination slot="hooper-addons" />
+    </Hooper>
+  </CoreFullscreen>
+
+</template>
+
+
+<script>
+
+  import _ from 'lodash';
+
+  import themeMixin from 'kolibri.coreVue.mixins.themeMixin';
+  import contentRendererMixin from 'kolibri.coreVue.mixins.contentRendererMixin';
+  import responsiveElement from 'kolibri.coreVue.mixins.responsiveElement';
+  import responsiveWindow from 'kolibri.coreVue.mixins.responsiveWindow';
+
+  import UiIconButton from 'kolibri.coreVue.components.UiIconButton';
+  import CoreFullscreen from 'kolibri.coreVue.components.CoreFullscreen';
+  import {
+    Hooper,
+    Slide,
+    Navigation as HooperNavigation,
+    Pagination as HooperPagination,
+  } from 'hooper';
+  import '../../styles/custom-hooper.css';
```

Comment:
let's move `custom-hooper.css` to the same directory as this component, since `SlideshowRendererComponent.vue` is the only place `Hooper` is used.

Also note that importing the css file like this will drop the styles into the global scope.

I don't know for sure, but we might be able scope it using something like...

```html
<style scoped>
  /deep/ .slideshow-renderer {
    @import './custom-hooper.css';
  }
</style>
```

or some other variation of sass-y goodness

Reply:
I refactored to import using CSS and moved the file. However I had issues when trying to import the CSS using `/deep/`  - since I have `<style scoped>` shouldn't that ensure the Hooper CSS is only used on this component?

Reply:
yes if it works without `/deep/` that's great. Sometimes I've encountered issues where that wasn't the case